extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -602,7 +611,7 @@ private function getOtherLanguages( Title $title ) {
 				'articleTitle' => str_replace( '_', ' ', $articleTitle ),
 				'url' => $url,
 			];
-		}, array_keys( $links ), array_values( $links ) );
+		} , array_keys( $links ), array_values( $links ) );
```

Comment:
@Warkot script from guidelines put it there - do you still think we should fix it?